commit 26f08e650748bb27247d4d646bcd95061bcd696f
Author: Trent Mick <trentm@gmail.com>
Date:   2019-09-12T13:29:21-07:00 (4 weeks ago)
    
    TOOLS-2254 add redirects from mo-docs to modocs.joyent.us (restdown files should now always be .md)

diff --git a/Makefile b/Makefile
index d26d68b..51b10e1 100644
--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@ TAP		:= ./node_modules/.bin/tap
 #
 # Files
 #
-DOC_FILES	 = index.restdown
+DOC_FILES	 = index.md
 JS_FILES	:= $(shell ls *.js) $(shell find lib test -name '*.js')
 JSL_CONF_NODE	 = tools/jsl.node.conf
 JSL_FILES_NODE   = $(JS_FILES)
diff --git a/docs/index.restdown b/docs/index.md
similarity index 100%
rename from docs/index.restdown
rename to docs/index.md
